DataAccess/Repositories/TargetAudienceRepository.cs

```diff
@@ -41,4 +42,23 @@ public void Delete(Guid id)
             repository.SaveChanges();
         }
     }
+
+    public List<TargetAudienceProjection> GetAllWithTestAmount()
+    {
+        using (Repository repository = new Repository())
+        {
+            var result = repository.TargetAudiences.Where(ta => ta.Id != Guid.Empty)
+                .Select(ta => new TargetAudienceProjection
+                {
+                    Label = ta.Label,
+                    From = ta.From.ToString(),
+                    To = ta.To.ToString(),
+                    AmountOfTests = ta.Tests.Count()
+                })
+                .ToList();
+
+            return result;
```

Comment:
Dan moet je hier toch wel die default id uithalen? (of wordt dat id van 0000-00000-0000-000 etc. gezien als empty?)